react/SelectBox/styles.styl

```diff
@@ -7,6 +8,13 @@
     height 0
     overflow hidden
 
+.select--autowidth
```

Comment:
Why not adding this `max-width` by default and removing it when you want fullwidth ?

Reply:
Actually I need to use a CSS class. Otherwise it's the customStyle from SelectBox which always have priority, I don't figured out why.